include/fastdds/dds/builtin/common/Types.hpp

```diff
@@ -34,7 +32,7 @@ namespace rpc {
 
 using InstanceName =
 #if FASTCDR_VERSION_MAJOR == 1
-        fastrtps::string_255;
+        fastcdr::string_255;
 #else
         fastcdr::string_255;
 #endif // FASTCDR_VERSION_MAJOR == 1
```

Comment:
Absolutely, good idea! But since it is also related to other header files and part of the code, I would open a different pr. Let me know what you think

Reply:
My intention was to just remove it in this specific place. But it is true that we should remove it also in other places and that should be done in a specific PR.